cmake/modules/SwiftSharedCMakeConfig.cmake

```diff
@@ -68,9 +68,9 @@ macro(swift_common_standalone_build_config_llvm product is_cross_compiling)
     set(${product}_NATIVE_LLVM_TOOLS_PATH "${LLVM_TOOLS_BINARY_DIR}")
   endif()
 
-  find_program(SWIFT_TABLEGEN_EXE "llvm-tblgen" "${${product}_NATIVE_LLVM_TOOLS_PATH}"
+  find_program(LLVM_TABLEGEN_EXE "llvm-tblgen" "${${product}_NATIVE_LLVM_TOOLS_PATH}"
```

Comment:
Is this going to step on any variables that LLVM sets up in an in-tree build configuration? (Not that that's officially supported at the moment, but I know of some people using it.)

I assume the answer is "no, this is safe", but there's an outside chance you'll say "oh, sure, I guess we can just use LLVM's variable".